README.md

```diff
@@ -28,35 +31,16 @@ You can now install the Slack client:
 composer require jolicode/slack-php-api
 ```
 
-## Usage
-
-First, you need to retrieve a token from Slack. 
-
-Checkout Slack's documentation about [all different kind of tokens](https://api.slack.com/authentication/token-types).
-A good starting point is the [Authentication Basics documentation](https://api.slack.com/authentication/basics).
-
-Then, use the factory that is provided to create the client:
+## Quick start
 
 ```php
 // $client contains all the methods to interact with the API
-$client = JoliCode\Slack\ClientFactory::create($yourToken);
+$client = JoliCode\Slack\ClientFactory::create($yourSlackToken);
 
 $user = $client->usersInfo(['user' => 'U123AZER'])->getUser();
-
-dump($user);
 ```
 
-<p align="center">
-  <img src="doc/images/model-sample.png" alt="Sample user object" />
-<p>
```

Comment:
Having some way to display the result in the quick start was great. Maybe we could show a "var_dump" output without image?

Reply:
I don't find this really useful and rather too big in the main readme. I expect someone using the Quick start only to explore by itself the code with its favorite editor